src/PhpCs/FiveLab/Sniffs/Namespace/NamespaceSniff.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/*
+ * This file is part of the FiveLab CiRules package
+ *
+ * (c) FiveLab
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code
+ */
+
+declare(strict_types = 1);
+
+namespace FiveLab\Component\CiRules\PhpCs\FiveLab\Sniffs\Namespace;
+
+use FiveLab\Component\CiRules\PhpCs\FiveLab\ErrorCodes;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+class NamespaceSniff implements Sniff
+{
+    public function register(): array
+    {
+        return [T_NAMESPACE];
+    }
+
+    public function process(File $phpcsFile, $stackPtr): void
+    {
+        $tokens = $phpcsFile->getTokens();
+        $endToken = $phpcsFile->findNext(T_SEMICOLON, $stackPtr);
+
+        if (false === $endToken) {
+            return;
+        }
+
+        $declaredNamespace = '';
+        for ($i = $stackPtr + 1; $i < $endToken; $i++) {
```

Comment:
Must be one blank line before logical construction (if, for, try/catch, etc...). And please check this moment below.